test/Crowdsale.js

```diff
@@ -1,5 +1,8 @@
+import moment from 'moment'
```

Comment:
`moment.js` it's still useful for calculating e.g. the number of seconds in a week:
```
moment.duration(1, 'week')
```
I may try to extract all of the usages to `increaseTime.js` and replace with a constant `WEEK`. What do you think about it?